src/main/java/com/domicilios/bugsbunny_api/model/entity/Usuario.java

```diff
@@ -0,0 +1,50 @@
+package com.domicilios.bugsbunny_api.model.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "usuario")
+@ToString
+
+public class Usuario {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter @Setter @Column(name = "id")
+    private Long id;
+
+    @Getter @Setter @Column(name = "nombres")
+    private String nombres;
+
+    @Getter @Setter @Column(name = "apellidos")
+    private String apellidos;
+
+    @Getter @Setter @Column(name = "email")
+    private String email;
+
+    @Getter @Setter @Column(name = "password")
+    private String password;
+
+    @Getter @Setter @Column(name = "celular")
+    private String celular;
+
+    @Getter @Setter @Column(name = "fecha_nacimiento")
+    private Date fechaNacimiento;
+
+    @Getter @Setter @Column(name = "genero")
+    private String genero;
+
+    @Getter @Setter @Column(name = "fecha_registro")
+    private Date fecha_registro;
```

Comment:
```suggestion
  @CreationTimestamp
    @Temporal(TemporalType.TIMESTAMP)
    @Column(name = "fecha_registro", updatable = false)
```

Como esta fecha es la fecha de registro del usuario no debemos ponerla nosotros, 
la debe poner la DB, y tampoco la podemos actualizar 

para que la ponga la BD, usamos `@CreationTimestamp
    @Temporal(TemporalType.TIMESTAMP)`
    
    Y para que no la podamos actualizar usamos `updatable = false`